### DISCUSSION TITLE
How to use LTR and RTL at the same time?

### DISCUSSION THREAD

**Discussion by ibrahimAboelsuod** [twbs/bootstrap#36170] [Q&A]
I am trying to build a website where the user can **switch** between LTR & RTL.
I could be missing something very basic but the docs are not clear about this. [here](https://getbootstrap.com/docs/5.1/getting-started/rtl/#ltr-and-rtl-at-the-same-time)

Just including the following doesn't seem to do anything but break the styles !
```
/* rtl:begin:options: {
  "autoRename": true,
  "stringMap":[ {
    "name": "ltr-rtl",
    "priority": 100,
    "search": ["ltr"],
    "replace": ["rtl"],
    "options": {
      "scope": "*",
      "ignoreCase": false
    }
  } ]
} */
.ltr {
  @import '~bootstrap/scss/bootstrap';
}
/*rtl:end:options*/
```

It feels like there is a missing step before this? What am I missing?

**Update:** One of the things I was missing was to run the postcss scripts, now this allows the import only in RTL and I get nothing in LTR.

**Comment by ffoodd** [comment]
If I understand, you need your end-users to be able switch by themselves, eg. change from english to arabic?

If so, you don't need to follow this snippet (meant to have both versions in the same page): you can simply switch `bootstrap.min.css` to `bootstrap.rtl.min.css` and vice-versa, depending on your users' setting.
